### ISSUE TITLE
macOS: feature directory missing

### ISSUE THREAD

**Issue by cederom** [ervandew/eclim#542]
While installing on macOS 10.12.6 I got this error that features directory is missing inside Eclipse home folder (`$HOME/eclipse/oxygen/Eclipse.app/Contents/Eclipse). After this directory is created by hand installation goes further..

Version: Oxygen.1a Release (4.7.1a)
Build id: 20171005-1200

```
bash-3.2$ java -version
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)

bash-3.2$ java -Dvim.files=$HOME/.vim -Declipse.home=$HOME/eclipse/oxygen/Eclipse.app/Contents/Eclipse/ -jar eclim_2.7.0.jar 
Buildfile: installer.xml
2017-10-24 11:47:19,726 INFO  [org.formic.Installer] Running Installer...
2017-10-24 11:47:29,618 INFO  [org.formic.util.CommandExecutor] which mvim
2017-10-24 11:47:29,659 INFO  [org.eclim.installer.step.VimStep] which mvim: /opt/local/bin/mvim
2017-10-24 11:47:29,675 INFO  [org.formic.util.CommandExecutor] vim -f -X -u NONE -U NONE --cmd redir! > /var/folders/jz/7xx01ppx1f9_g52ydds26fzc0000gn/T/eclim_installer8280376594791056284.tmp | silent! echo &rtp | quit
2017-10-24 11:47:46,358 WARN  [org.eclim.installer.step.RequirementProvider] Error checking eclipse version via plugins.
2017-10-24 11:47:46,395 INFO  [org.formic.util.CommandExecutor] vim --version
2017-10-24 11:48:01,122 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclipse.equinox.p2.director -repository file:///var/folders/jz/7xx01ppx1f9_g52ydds26fzc0000gn/T/formic_1143666422/update -installIU org.eclim.installer.feature.group
2017-10-24 11:48:12,328 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclim.installer.application -info
2017-10-24 11:48:19,214 INFO  [org.eclim.installer.step.EclipseInfo]   Profile: _Users_cederom_eclipse_oxygen_Eclipse.app_Contents_Eclipse
2017-10-24 11:48:19,214 INFO  [org.eclim.installer.step.EclipseInfo] eclipse.profile=_Users_cederom_eclipse_oxygen_Eclipse.app_Contents_Eclipse
2017-10-24 11:48:19,214 INFO  [org.eclim.installer.step.EclipseInfo]   Configuration: file:/Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/configuration/
2017-10-24 11:48:19,214 INFO  [org.eclim.installer.step.EclipseInfo] eclipse.local=/Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse
2017-10-24 11:48:19,216 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.cdt 9.3.2.201709131603 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,217 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.help 2.2.101.v20171009-0410 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.bugzilla_feature 3.23.1.v20170623-0008 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.egit 4.8.0.201706111038-r file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn_feature 3.23.1.v20170623-0008 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse 3.7.3.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.local 3.7.0.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.commons.identity 1.15.0.v20170411-1844 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.oomph.p2 1.8.0.v20170410-0909 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.tm.terminal.view.feature 4.3.0.201706140544 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,218 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.commons.notifications 1.15.0.v20170411-1844 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rcp 4.7.1.v20171009-0410 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.epp.package.cpp.feature 4.7.1.20171005-1200 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.github.feature 4.6.0.201612231935-r file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.cdt.mylyn 5.19.0.v20170411-2108 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.emf.common 2.13.0.v20170609-0707 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.egit.mylyn 4.6.1.201703071140-r file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.oomph.setup 1.9.0.v20170706-0615 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.emf.ecore 2.13.0.v20170609-0707 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,219 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.ssh 3.7.0.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.jgit 4.8.0.201706111038-r file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.wikitext_feature 3.0.6.201703111926 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.platform 4.7.1.v20171009-0410 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.discovery 3.23.0.v20170411-1844 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.wikitext.editors_feature 3.0.6.201703111926 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.wst.xml_ui.feature 3.8.2.v201702270442 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.tasks.ide 3.23.1.v20170623-0008 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,220 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.userstorage 1.1.0.v20170526-1605 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.cdt.build.crossgcc 9.3.2.201709131603 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.commons 3.23.0.v20170503-0014 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.ide_feature 3.23.0.v20170411-2108 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.epp.mpc 1.6.2.v20171004-2006 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.dstore 3.7.0.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.cdt.gdb 9.3.2.201709131603 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.team_feature 3.23.0.v20170411-2108 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,221 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.commons.repositories 1.15.0.v20170411-1844 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,227 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.ftp 3.7.1.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,227 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.telnet 3.7.0.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,228 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.monitor 3.23.0.v20170411-1844 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,228 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.tm.terminal.control.feature 4.3.0.201706140544 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,228 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.tm.terminal.feature 4.3.0.201706140544 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,228 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.cdt.launch.remote 9.3.2.201709131603 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,228 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclim.installer 2.7.0 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,231 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.rse.core 3.7.3.201704251225 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,232 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.mylyn.context_feature 3.23.0.v20170414-0629 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:19,232 INFO  [org.eclim.installer.step.EclipseInfo]   Feature: org.eclipse.oomph.setup.core 1.8.0.v20170531-0903 file:/Users/cederom/.p2/pool/
2017-10-24 11:48:45,752 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclipse.equinox.p2.director -repository http://pydev.org/updates -list
2017-10-24 11:49:00,548 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclim.installer.application -repository http://pydev.org/updates -installIU org.python.pydev.feature.feature.group
2017-10-24 11:49:14,346 INFO  [ANT] [eclim:shutdown] Unable to shut down eclimd (9091): java.lang.NullPointerException - null
2017-10-24 11:49:14,604 INFO  [ANT]      [exec] rm: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/eclim: No such file or directory
2017-10-24 11:49:14,914 ERROR [ANT]      [exec] Result: 1
2017-10-24 11:49:15,033 INFO  [ANT]      [exec] rm: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/eclimd: No such file or directory
2017-10-24 11:49:15,340 ERROR [ANT]      [exec] Result: 1
2017-10-24 11:49:15,375 ERROR [ANT] 
jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:342: Directory does not exist: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/features
        at org.apache.tools.ant.taskdefs.Delete.handle(Delete.java:704)
        at org.apache.tools.ant.taskdefs.Delete.execute(Delete.java:613)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
        at org.apache.tools.ant.Project.executeTargets(Project.java:1251)
        at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:458)
        at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:119)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.Project.executeTarget(Project.java:1368)
        at org.formic.wizard.step.shared.InstallAction.execute(InstallAction.java:124)
        at org.formic.wizard.step.gui.InstallStep$1.run(InstallStep.java:162)
        at foxtrot.AbstractWorkerThread$1.run(AbstractWorkerThread.java:40)
        at java.security.AccessController.doPrivileged(Native Method)
        at foxtrot.AbstractWorkerThread.runTask(AbstractWorkerThread.java:36)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:199)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:182)
        at java.lang.Thread.run(Thread.java:748)
2017-10-24 11:49:15,379 ERROR [ANT] 
jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:145: The following error occurred while executing this line:
jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:342: Directory does not exist: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/features
        at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:551)
        at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:460)
        at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:119)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.Project.executeTarget(Project.java:1368)
        at org.formic.wizard.step.shared.InstallAction.execute(InstallAction.java:124)
        at org.formic.wizard.step.gui.InstallStep$1.run(InstallStep.java:162)
        at foxtrot.AbstractWorkerThread$1.run(AbstractWorkerThread.java:40)
        at java.security.AccessController.doPrivileged(Native Method)
        at foxtrot.AbstractWorkerThread.runTask(AbstractWorkerThread.java:36)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:199)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:182)
        at java.lang.Thread.run(Thread.java:748)
Caused by: jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:342: Directory does not exist: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/features
        at org.apache.tools.ant.taskdefs.Delete.handle(Delete.java:704)
        at org.apache.tools.ant.taskdefs.Delete.execute(Delete.java:613)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
        at org.apache.tools.ant.Project.executeTargets(Project.java:1251)
        at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:458)
        ... 19 more
jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:145: The following error occurred while executing this line:
jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:342: Directory does not exist: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/features
        at org.apache.tools.ant.ProjectHelper.addLocationToBuildException(ProjectHelper.java:551)
        at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:460)
        at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:119)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.Project.executeTarget(Project.java:1368)
        at org.formic.wizard.step.shared.InstallAction.execute(InstallAction.java:124)
        at org.formic.wizard.step.gui.InstallStep$1.run(InstallStep.java:162)
        at foxtrot.AbstractWorkerThread$1.run(AbstractWorkerThread.java:40)
        at java.security.AccessController.doPrivileged(Native Method)
        at foxtrot.AbstractWorkerThread.runTask(AbstractWorkerThread.java:36)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:199)
        at foxtrot.workers.SingleWorkerThread.run(SingleWorkerThread.java:182)
        at java.lang.Thread.run(Thread.java:748)
Caused by: jar:file:/Users/cederom/Downloads/eclim_2.7.0.jar!/installer.xml:342: Directory does not exist: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/features
        at org.apache.tools.ant.taskdefs.Delete.handle(Delete.java:704)
        at org.apache.tools.ant.taskdefs.Delete.execute(Delete.java:613)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:291)
        at sun.reflect.GeneratedMethodAccessor13.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:390)
        at org.apache.tools.ant.Target.performTasks(Target.java:411)
        at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1399)
        at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
        at org.apache.tools.ant.Project.executeTargets(Project.java:1251)
        at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:458)
        ... 19 more
Installer Context (Please include in bug reports):
  eclipse.home=/Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse
  eclipse.info=org.eclim.installer.step.EclipseInfo@36b0c039
  eclipse.local=/Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse
  eclipse.profile=_Users_cederom_eclipse_oxygen_Eclipse.app_Contents_Eclipse
  featureList.adt=false
  featureList.cdt=true
  featureList.dltk=false
  featureList.dltkruby=false
  featureList.groovy=false
  featureList.jdt=false
  featureList.pdt=false
  featureList.pydev=true
  featureList.sdt=false
  featureList.wst=false
  vim.files=/Users/cederom/.vim
  vim.skip=false
2017-10-24 11:50:24,633 INFO  [org.formic.Installer] Installer Finished.
2017-10-24 11:50:24,635 DEBUG [ANT] 
BUILD SUCCESSFUL
Total time: 3 minutes 5 seconds
2017-10-24 11:50:24,646 WARN  [ANT]      [echo] Installation canceled.
2017-10-24 11:50:24,666 WARN  [ANT]      [echo] 
2017-10-24 11:50:24,667 WARN  [ANT]      [echo] ##
2017-10-24 11:50:24,667 WARN  [ANT]      [echo] # Please wait while the eclim installer feature is removed from eclipse...
2017-10-24 11:50:24,667 WARN  [ANT]      [echo] ##
2017-10-24 11:50:24,667 WARN  [ANT]      [echo]         
2017-10-24 11:50:24,669 INFO  [ANT] [eclim:uninstalliu] Uninstalling org.eclim.installer...
2017-10-24 11:50:24,670 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclipse.equinox.p2.director -repository file:///var/folders/jz/7xx01ppx1f9_g52ydds26fzc0000gn/T/formic_1143666422/update -uninstallIU org.eclim.installer.feature.group


2017-10-24 11:50:34,072 INFO  [org.eclim.installer.ant.UninstallIUTask] Uninstalling org.eclim.installer.feature.group 2.7.0.
^C^C2017-10-24 11:50:34,970 WARN  [org.eclim.installer.ant.UninstallIUTask] error:  out: Uninstalling org.eclim.installer.feature.group 2.7.0.
2017-10-24 11:50:34,971 INFO  [ANT] [eclim:uninstalliu] Invoking p2 gc...
2017-10-24 11:50:34,971 INFO  [org.formic.util.CommandExecutor] java -Xmx256m -Dhttp.nonProxyHosts=local|*.local|169.254/16|*.169.254/16 -jar /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -clean -application org.eclipse.equinox.p2.garbagecollector.application -profile _Users_cederom_eclipse_oxygen_Eclipse.app_Contents_Eclipse
^C^C2017-10-24 11:50:36,103 WARN  [org.eclim.installer.ant.UninstallIUTask] error:  out: 
2017-10-24 11:50:36,104 INFO  [org.eclim.installer.ant.UninstallIUTask] Removing temp repositories from p2 prefs: /Users/cederom/eclipse/oxygen/Eclipse.app/Contents/Eclipse/p2/org.eclipse.equinox.p2.engine/profileRegistry/_Users_cederom_eclipse_oxygen_Eclipse.app_Contents_Eclipse.profile/.data/.settings/org.eclipse.equinox.p2.metadata.repository.prefs
2017-10-24 11:50:36,104 WARN  [org.eclim.installer.ant.UninstallIUTask] P2 repository prefs file doesn't exist.
```

**Comment by GuutBoy** [comment]
I am having similar problems. I tried just creating the `features` dir. This gets the eclim installer a little further, but it then just hits an other problem. I also tried the advice given in in the comments to #479. I.e. running `java -Dvim.skip=true -Declipse.home=$HOME/eclipse/java-oxygen/Eclipse.app/Contents/Eclipse -Declipse.local=$HOME/.p2/pool -jar ~/Downloads/eclim_2.7.0.jar install` this gives yet an other error.
